packages/components/src/index.ts

```diff
@@ -71,15 +71,10 @@ export type { RenderThumbFunction } from './pricing-slider/types';
 // Logos
 export { default as ClientLogoList } from './client-logo-list';
 export { default as JetpackLogo } from './logos/jetpack-logo';
-export { default as BloombergLogo } from './logos/bloomberg-logo';
 export { default as CloudLogo } from './logos/cloud-logo';
 export { default as CNNLogo } from './logos/cnn-logo';
```

Comment:
Other corporate logos will be addressed separately as they require a bit more clean up.